docs/application-connector/docs/050-gs-calling-registered-service-from-kyma.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Calling registered service from within Kyma
+type: Getting Started
+---
+
+This guide shows how to call the registered service from within Kyma using a simple lambda function.
+
+
+## Prerequisites
+
+- Remote Environment created and bound to the `production` Environment.
+- Client certificates for the RE generated.
+
+
+## Steps
+
+1. Register a service with the following specyfication to the desierd Remote Envoronment:
```

Comment:
Same comment as in the previous document. The sentence should look something like this: 

Register a simple service. with the following specification in a Remote Environment. 
>**NOTE:** To learn how to register a service, see the **Register a service** Getting Started Guide.